doc/devdocs/modules/launcher/new-plugin-checklist.md

```diff
@@ -0,0 +1,30 @@
+# New plugin checklist
+- A plugin is a project under `modules\launcher\Plugins`
+- Microsoft plugin project name pattern: `Microsoft.PowerToys.Run.Plugin.{PluginName}`
+- Community plugin project name pattern: `Community.PowerToys.Run.Plugin.{PluginName}`
+- [`GlobalSuppressions.cs`](/src/codeAnalysis/GlobalSuppressions.cs) and [`StyleCop.json`](/src/codeAnalysis/StyleCop.json) have to be included in the plugin project so it follows PowerToys code guidelines
+- A plugin has to have `{PowerToys version}.0` version
+- A plugin has to have only x64 platform target
+- A plugin has to contain a `plugin.json` file of the following format in its root folder
```

Comment:
suggest deep linking to a known good sample

Reply:
Can you elaborate? A scheme of `plugin.json` with comments is already included.